### PR TITLE
Add spec for /var/log/libvirt/libvirtd.log

### PR DESCRIPTION
**Pull request by psachin**
Signed-off-by: Sachin Patil <psachin@redhat.com>

### CODE REVIEW

**Comment by bfahr** [comment]
@psachin I believe these changes are OK, I'm investigating the CI failure.

**Comment by bfahr** [comment]
test this please

**Comment by patchkez** [comment]
test this please

File: uploader.json [hunk 1]
```diff
@@ -1930,6 +1930,17 @@
...
             "symbolic_name": "ksmstate"
         },
+        {
+            "file": "/var/log/libvirt/libvirtd.log",
+            "pattern": [
+		"virDomainBlockPull:",
+		"virDomainBlockCopy:",
+		"virDomainBlockCommit:",
+		"virDomainSnapshotCreateXML:",
+		"qemuMigrationSrcNBDStorageCopyBlockdev:",
```
Comment (patchkez) [inline uploader.json]: Comma at the end of line is extra, please remove it, I think it is the case also for uploader.v2.json. @bfahr

**Pull request merged**